ratis-shell/src/main/java/org/apache/ratis/shell/cli/RaftUtils.java

```diff
@@ -65,6 +65,10 @@ public static RaftPeerId getPeerId(String host, int port) {
    */
   public static RaftClient createClient(RaftGroup raftGroup) {
     RaftProperties properties = new RaftProperties();
+    // Since ratis-shell support GENERIC_COMMAND_OPTIONS, here we should
+    // set these options to raft properties to make it work.
+    System.getProperties().stringPropertyNames().forEach(
+        key -> properties.setIfUnset(key, System.getProperty(key)));
     RaftClientConfigKeys.Rpc.setRequestTimeout(properties,
```

Comment:
@AngersZhuuuu , thanks for working on this!  This is a good change.  We should check if the requestTimeout is already set.

Reply:
> We should check if the requestTimeout is already set.

How about the current change? Properties in `System` should have higher priority. It can cover the requestTimeout if we set it passed by `-D`